.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,3 +19,15 @@ the linked issue.
 -->
 
 Related to #
+
+#### Things to check
+
+Complete all of the items on this checklist **before** merging the PR into the trunk. Both the PR author and review are responsible for ensuring the checklist is completed. 
+
+Not all items will apply to every PR, and any inapplicable items can just be marked as complete.
+
+- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
+- [ ] Have you tested the feature in [Simple](https://atomicuniversity.wordpress.com/what-is-atomic/make-a-site-atomic/simple-vs-atomic), [Atomic](https://atomicuniversity.wordpress.com/what-is-atomic), and [self-hosted Jetpack sites](https://fieldguide.automattic.com/jurassic-ninja)?
+- [ ] Are there any TypeScript or console errors?
+- [ ] Are we memoizing when appropriate (for expensive computations)?
+- [ ] Have we sent any new strings [for translation](https://fieldguide.automattic.com/i18n/how-translation-projects-work) ASAP?
```

Comment:
Since it's for Calypso, explaining just the usage of `[Status] String Freeze` might be the best solution. The original announcement post (p4TIVU-5Jq-p2) has a pretty good description of the process, so maybe it could be added to calypso docs, with some small updates/corrections?